File: Advanced/BottomNavigationViewDemo/app/src/main/java/edu/cs4730/bottomnavigationviewdemo/threeFragment.java
package edu.cs4730.bottomnavigationviewdemo;


import android.os.Bundle;

import androidx.annotation.NonNull;
import androidx.fragment.app.Fragment;

import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;

import edu.cs4730.bottomnavigationviewdemo.databinding.FragmentThreeBinding;


/**
 * A simple {@link Fragment} subclass.
 */
public class threeFragment extends Fragment {
    FragmentThreeBinding binding;

    public threeFragment() {
        // Required empty public constructor
    }


    @Override
    public View onCreateView(@NonNull LayoutInflater inflater, ViewGroup container,
                             Bundle savedInstanceState) {
        // Inflate the layout for this fragment
       binding = FragmentThreeBinding.inflate(inflater, container, false);
       return binding.getRoot();
    }

}

File: Advanced/BottomNavigationViewDemo_kt/app/src/main/java/edu/cs4730/bottomnavigationviewdemo_kt/threeFragment.kt
package edu.cs4730.bottomnavigationviewdemo_kt

import android.view.LayoutInflater
import android.view.ViewGroup
import android.os.Bundle
import android.view.View
import androidx.fragment.app.Fragment
import edu.cs4730.bottomnavigationviewdemo_kt.R

/**
 * A simple [Fragment] subclass.
 */
class threeFragment : Fragment() {
    override fun onCreateView(
        inflater: LayoutInflater, container: ViewGroup?,
        savedInstanceState: Bundle?
    ): View? {
        // Inflate the layout for this fragment
        return inflater.inflate(R.layout.fragment_three, container, false)
    }
}